soundmachine.html

```diff
@@ -0,0 +1,27 @@
+<html>
+    <head>
+        <title>Sound Machine</title>
+    </head>
+    <body>
+        
+    <script>
+      function play() {
+        var audio = document.getElementById("audio");
+        audio.play();
+      }
+    </script>
+
+        <input type="button" value="PLAY" onclick="play()">
+        <audio id="audio" src=""></audio></br>
+        
+        <button type="button" value="">Sound #1</button>
+        <button type="button" value="">Sound #2</button>
+        <button type="button" value="">Sound #3</button></br>
+        <button type="button" value="">Sound #4</button>
+        <button type="button" value="">Sound #5</button>
+        <button type="button" value="">Sound #6</button></br>
+        <button type="button" value="">Sound #7</button>

```

Comment:
is type ="button" necessary if using the button html tag?

Reply:
I think so.. one of 3 types